css/backoffice/_shame.scss

```
@@ -43,3 +43,9 @@
 .ibo-svg-illustration--container > svg *[fill="#6c63ff"]{
 	fill: $ibo-svg-illustration--fill;
 }
+
+// Workaround for a Safari (unsupported browser) issue: https://sourceforge.net/p/itop/discussion/itop-hub/thread/fc97ec4a10/?limit=25#1cc0/9930
+// Fix found here: https://stackoverflow.com/questions/40895387/z-index-not-working-on-safari-fine-on-firefox-and-chrome
+.ibo-navigation-menu.ibo-is-active .ibo-navigation-menu--drawer{
+    -webkit-transform: translate3d(0,0,0);
```

Not sure if WebKit needs to be addressed specifically, I think this is also good:
```suggestion
    transform: translate3d(0,0,0);
```

---

Webkit need not to be addressed specifically (transform works as well), but the idea was to address specifically since there is no need for this change i.e. for firefox

---

No need, but also doesn't harm. Next to that I think the `-webkit` prefix is also relevant for Chrome, Edge and Opera.

---

think you are right Chrome and Edge impacted by `-webkit` as well and I am with this change doesn't harm.
But see you any benefit in using `transform` instead of `-webkit-transform`?

---

Using web standards instead of property implementations? Also that would eliminate possible future FF only bug by not having this specific directive for that browser. (So all browsers have the same settings for these classes)

---

Is `-webkit-transform` a proprietary implementations?
Anyhow I am absolutely fine with your suggestion but would be interested in @piRGoif or @Molkobain opinion here?

---

Any browser prefixed one is a proprietary implementation that might or might not follow the standard. Mostly that is because they exist before it became a standard..

---

Hello,
Indeed I'm not sure what would be the right decision here... We already tried to follow web standards as best as we can considering the legacy code, but a bug like this happened anyway...

I'm more inclined with Lars way of thinking (if it works, don't upgrade ;) ), but I understand your point Thomas (same code for all of the browsers)

We'll debate what would be the best option during the technical review !

---

It's fine by me as long as it still works as intended on Firefox (the only non-webkit based supported browsers).
Also, did you went through most of the backoffice GUI to make sure it did not introduce any glitch on other parts such as modals, datatables etc?

---

@larhip ? Functional review is next week.

---

PS: Generic property would be great if no impact on FF / Chrome.

---

So I tested both possibilities webkit and non-webkit implementations, and checke on Safari, Chrome and Firefox not only the menu but also the detail views, list views, ...

I was not able to see any difference using webkit or not, and I was also not able to see any glitches or something similar on any view.

> Accepted during technical review (pending the change / answer on the generic property instead of the webkit one)

When I got your feedback right you prefer the Webstandard way (without webkit)? 
Since this change seems to have no impact to firefox I would be fine with that as well.
Could you give me a brief feedback, so I could adapt it or not?

---

Yes property `transform` is preferred over `-webkit-transform` :)

---

Done ✅ 

> Edit: I might have messed something up in my comments, some of yesterday messages appear as of just now.

Thanks for the note I was already confused about conversation history :-)